### DISCUSSION TITLE
[BUG] Error evaluating 'gcode_macro _print_end_inner:gcode': jinja2.exceptions.UndefinedError: tuple object has no element 0

### DISCUSSION THREAD

**Discussion by mochoandre** [jschuh/klipper-macros#200] [Q&A] [answered]
**Describe the bug**
When I Start a print or do CHECK_KM_CONFIG I get 

`Error evaluating 'gcode_macro _print_end_inner:gcode': jinja2.exceptions.UndefinedError: tuple object has no element 0`

`Error evaluating 'gcode_macro check_km_config:gcode': jinja2.exceptions.UndefinedError: tuple object has no element 0`

[Voron 2.4 - Klipper_config - 231107.zip](https://github.com/jschuh/klipper-macros/files/13289759/Voron.2.4.-.Klipper_config.-.231107.zip)

`

**Comment by jschuh** [comment, marked as answer]
First, please ask questions here rather than filing bugs if you need help troubleshooting your config.

As for your config, you're setting `variable_print_min` and `variable_print_max` to empty tuples, which are illegal values. Either don't set them at all (so you get the defaults) or set them to appropriate values for your printer (the comment demonstrates the format).
